File: 백준/Bronze/2490. 윷놀이/윷놀이.swift
import Foundation

for _ in 0..<3 {
    let input = readLine()!.split(separator: " ").map{String($0)}
    var answer = 0 // 등(1)의 개수
    for i in input {
        if (i == "1") { answer += 1 }
    }
    
    if (answer == 0) { print("D") } // 윷
    else if (answer == 1) { print("C") } // 걸
    else if (answer == 2) { print("B") } // 개
    else if (answer == 3) { print("A") } // 도
    else if (answer == 4) { print("E") } // 모
}

File: 백준/Bronze/2490. 윷놀이/윷놀이.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        for (int i=0; i<3; i++) {
            String s = br.readLine();
            StringTokenizer st = new StringTokenizer(s);
            
            int answer = 0;
            for(int j=0; j<4; j++) {
                if (Integer.parseInt(st.nextToken()) == 1) answer++;
            }

            if (answer == 3) System.out.println("A");
            else if (answer == 2) System.out.println("B");
            else if (answer == 1) System.out.println("C");
            else if (answer == 0) System.out.println("D");
            else if (answer == 4) System.out.println("E");
        }
    }
}